linux/LICENSE.md

```diff
@@ -0,0 +1,4 @@
+The [kmflcomp](./kmflcomp) and [libkmfl](./libkmfl) projects are covered by the [MIT license](./libkmfl/COPYING)
+
+The [ibus-kmfl](./ibus-kmfl) project is licensed under GNU General Public License as published by the Free Software Foundation; either [version 2 of the License](./ibus-kmfl/COPYING), or (at your option) any later version.
+2 files in ibus-kmfl, [kmflutil.c](./ibus-kmfl/src/kmflutil.c) and [kmflutil.h](./ibus-kmfl/src/kmflutil.h) are covered by the MIT license and may be used as such in other projects.
```

Comment:
This needs to explicitly state that these two files are dual licensed MIT and GPL

Reply:
I don't understand why you want them to be dual licensed since MIT is GPL compatible

Reply:
It makes the intent clear. Without that, there is some confusion from other developers as to what we intend. With dual licensing, we are making it clear to developers that we understand what we are doing and not that we are just mixing and matching licenses and hoping for the best :)

Reply:
Done.